projects/aca-folder-rules/src/lib/rule-details/options/rule-options.ui-component.scss

```diff
@@ -11,4 +11,8 @@
   &.read-only .mat-checkbox-inner-container {
     display: none;
   }
+
+  .hide-error-script-dropdown {
+    opacity: 0;
```

Comment:
is there any reason for opacity compared to "display: none" ?

Reply:
I'm assuming this is to avoid scrolling down

Reply:
Yes, this is intentional. With `display: none`, the white-space would disappear, which means that when it's re-enabled, the user needs to scroll to see it, which is problematic since they may not realize it's there.